docs/unified-test-documentation/dasharo-compatibility/347-sign-of-life.md

```diff
@@ -0,0 +1,81 @@
+# Dasharo compatibility: Sign of life
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+## SOL001.001 Detection of Dasharo firmware and Dasharo EC firmware
+
+**Test description**
+
+This test aims to verify that the information about the version of Dasharo
+firmware and Dasharo EC firmware is recognized correctly and displayed during
+the boot phase.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `EC firmware` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Note the results.
```

Comment:
To make the test correspond to the planned automatic tests, we should add the information:

```
2. Wait for Sign of life string
```